extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -1126,6 +1126,20 @@ export class SchemaCompareMainWindow {
 		if (ownerUri) {
 			endpointInfo = endpoint;
 			endpointInfo.ownerUri = ownerUri;
+		} else if (endpoint.endpointType === mssql.SchemaCompareEndpointType.Project) {
+			endpointInfo = {
+				endpointType: endpoint.endpointType,
+				packageFilePath: '',
+				serverDisplayName: '',
+				serverName: '',
+				databaseName: '',
+				ownerUri: '',
+				connectionDetails: undefined,
+				projectFilePath: endpoint.projectFilePath,
+				targetScripts: [],
+				dataSchemaProvider: '',
+				folderStructure: loc.schemaObjectType			// TODO: Pick this automatically from the scmp file, after issue #20332 is resolved (check dsp as well)
```

Comment:
not for this PR, but I just noticed that the localized strings of  schemaObjectType and the the other folder structure strings are getting sent to DacFx. This might cause problems for non-English languages. Can you create a follow up task to check this? It would really be better to be sending an enum instead of a string anyway.

Reply:
Sure, I will create a task for it.

Reply:
Created this: https://github.com/microsoft/azuredatastudio/issues/20344